pages.ta/android/bugreportz.md

```diff
@@ -8,14 +8,14 @@
 
 `bugreportz`
 
-- இயங்கும் `bugreportz` செயல்பாட்டின் முன்னேற்றத்தைக் காட்டு:
+- இயங்கும் செயல்பாட்டின் [p] முன்னேற்றத்தைக் காட்டு:
```

Comment:
I don't think languages other than English need `[ ]` syntax. It's hard to relate a Latin character with them.

Reply:
Agreed, Will remove it. Because the native word doesn't signify a long option. I forgot and did it as usual for keybinding translations.

Reply:
> I don't think languages other than English need [ ] syntax. It's hard to relate a Latin character with them.

I like this idea, but what about Russian language? There are similar looking characters. We should define a formal rule for what languages omitting square brackets is permitted.